editoast/src/views/study.rs

```diff
@@ -103,7 +111,7 @@ impl StudyCreateForm {
             .state(self.state)
             .study_type(self.study_type)
             .project_id(project_id);
-        Study::validate(&study_changeset)?;
+        Study::validate(&study_changeset).map_err(Into::<StudyError>::into)?;
```

Comment:
Isn't it equivalent to the following?


```suggestion
        Study::validate(&study_changeset).map_err(StudyError::from)?;
```

Reply:
yes, thank you!